### DISCUSSION TITLE
lab4-1跑 source run_sim時發生錯誤2.0

### DISCUSSION THREAD

**Discussion by ghost** [bol-edu/HLS-SOC-Discussions#130] [General]
我想請教一下，我們在跑source run_sim一樣會有下面error(圖1)，我有照先前發問的回答將counter_la_fir.c中的#include <stub.c>拿掉，也一樣會有這個error(有加沒加都有)，想請問解決的方法。
然後我有單獨跑$ riscv32-unknown-elf-gcc -I../../firmware -o counter_la_fir.elf ..這個指令(圖二)，但我發現資料夾內沒有產生對應的counter_la_fir.elf(圖三)。不知道是什麼原因。
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/119398292/673ca092-f0b1-4eba-b5ca-71e59fd16840)
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/119398292/f9287f6d-ea24-40b7-8856-7eb4eb9c18e5)
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/119398292/b8a4f624-f0f0-4b89-ad73-7424642a6078)

**Comment by yihsintsai1003** [comment]
你的user_proj_example.counter.v可能有問題。

**Reply by ghost** [reply]
感謝你的幫忙，我順利找到原因並且解決了，的確是user_proj_example.counter.v有問題

**Reply by yui0303** [reply]
你好，可以請問最後怎麼解決的嗎，我們這組好像遇到一樣的問題，謝謝！

**Reply by ghost** [reply]
user_proj_example.counter.v的clock與reset沒接上WB的
